src/plone/restapi/services/addons/addons.py

```diff
@@ -2,16 +2,22 @@
 from Products.CMFCore.utils import getToolByName
 from Products.CMFPlone import PloneMessageFactory as _
 from Products.CMFPlone.interfaces import INonInstallable
-from Products.CMFQuickInstallerTool.interfaces import (
```

Comment:
Uhm, this is strange, the iface was moved to:
```
from Products.CMFPlone.interfaces import INonInstallable
```
And we should have aliases for that:

https://github.com/plone/plone.app.upgrade/blob/ef4aef834c561f7a634e819571b7c3bfb48cf314/plone/app/upgrade/__init__.py#L185

Don't you have plone.app.upgrade?

I think the proper fix is to replace the import given that the alias is there since 5.1:
https://github.com/plone/Products.CMFPlone/blob/5.1.x/Products/CMFPlone/interfaces/__init__.py#L38
and this package supports Plone 5.2 only.

Also the towncrier entry should be updated accordingly

Reply:
No I do only depend on `Products.CMFPlone`. Which is totally fine. `plone.app.upgrade` in only installed in a single service instance. The cluster runs usually without.

Reply:
> I think the proper fix is to replace the import given that the alias is there since 5.1:
Right, this the current master does not support 5.1 anyway. Removal is probably the best.